modules/tinymce/CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Search and replace plugin would incorrectly find matching text inside non-editable root elements. #TINY-10162
 - Search and replace plugin would incorrectly find matching text inside SVG elements. #TINY-10162
 - Removed use of `async` for editor rendering which caused visual blinking when reloading the editor in-place. #TINY-10249
+- List items containing the first and the last element as text and a nested list caused some list operations to fail. #TINY-10268
```

Comment:
Maybe not sure if its better:
```suggestion
- List items containing a list element surrounded by non list nodes would cause some list operations to fail. #TINY-10268
```
Writing changelog items is harder than naming things. :smile:

Reply:
changed